docs/developer-docs/quickstart/create_first_dapp.md

```diff
@@ -0,0 +1,59 @@
+# Create a Dapp
+
+## Setup
+
+First, let’s make sure that your development environment is ready.
+
+* You’ll be using your own text editor and terminal app for this tutorial.
+* If you don’t have the Internet Computer SDK installed, install it
+  from [here](https://internetcomputer.org/docs/current/developer-docs/build/install-upgrade-remove), or keep following
+  along below.
+
+### Install the Internet Computer SDK
+
+```bash
+ sh -ci "$(curl -fsSL https://internetcomputer.org/install.sh)"
+```
+
+:::tip
+If you are on Windows, we recommend downloading [Git for Windows](https://gitforwindows.org/) and using Git Bash, which
+supports the UNIX-specific commands in this tutorial. You can also
+use [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
+:::
+
+## Create Project
+
+To create an IC dapp, open your terminal, cd into the directory you’d like to create the app in, and run the following
+command:
+
+```bash
+dfx new hello
+```
+
+:::info
+Under the hood, dfx bootstraps a new project for you using a blank project template. If it doesn’t work, please visit
+the [developer forum](https://forum.dfinity.org).
+:::
+
+## Deploy
+
+You now have a new directory called `hello`. Let’s cd into it:
+
+```bash
+cd hello
+```
+
+From here, we’ll deploy to the Internet Computer by using the Playground, which will allow you to deploy your app free
+of charge for a short period of time.
```

Comment:
Do we want to indicate what "short period of time" means? I immediately find myself asking this :)

Reply:
Thanks for the suggestion @domwoe. Good idea, I'll update.